test/dlc_tests/eks/tensorflow/inference/test_eks_tensorflow_inference.py

```diff
@@ -14,13 +14,13 @@ def test_eks_tensorflow_half_plus_two_inference(tensorflow_inference):
 
     processor = "gpu" if "gpu" in tensorflow_inference else "cpu"
 
-    model_name = f"saved_model_half_plus_two_{processor}"
+    model_name = f"albert"
```

Comment:
the name of this function is test_eks_half_plus_two_inference -- do you want to change it completely? or add a new bert model test?

Reply:
+1